usage.c

```diff
@@ -15,7 +15,12 @@ void vreportf(const char *prefix, const char *err, va_list params)
 
 	fflush(fh);
 	if (!tweaked_error_buffering) {
+#if defined(_MSC_VER)
+		/* UCRT doesn't like zero buffer size */
+		setvbuf(fh, NULL, _IOLBF, BUFSIZ);
+#else
 		setvbuf(fh, NULL, _IOLBF, 0);
+#endif
```

Comment:
(I just found out that it is a BSD compatibility function, so maybe it is not such a good idea...)